### ISSUE TITLE
can we expose BaM nvme driver without HBM cache capability to python world directly?

### ISSUE THREAD

**Issue by gaowayne** [ZaidQureshi/bam#48]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
I would like save tensor myself with BaM on nvme directly. no need cache logic from BaM
**Describe alternatives you've considered**

**Additional context**

**Comment by msharmavikram** [comment]
No. HBM cache is not a cache. It is mainly an IO buffer. What you can do is reduce the cache size to a tiny amount, perhaps 128MB. The BaM will work. However, remember you might end up getting a lot of cache thrashing with a tiny cache.
